csharp/protos/extensions/extensions_a.proto

```diff
@@ -0,0 +1,12 @@
+syntax = "proto3";
```

Comment:
naming: for .proto files that are there to test for previously reported issues, a pattern like this is used:
https://github.com/protocolbuffers/protobuf/blob/master/csharp/protos/unittest_issues.proto

because we need 3 different .proto files, perhaps we could name them `unittest_issue6936_a.proto` or sth like that?

Reply:
Sure. Should it use the `unittest_issues` package and namespace as well?